app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
```

Comment:
You can simplify the import statement by directly importing `datetime` without specifying the module, or alternatively, you should directly import the specific method you will be using.